themes/default-bootstrap/js/order-opc.js

```diff
@@ -924,7 +924,7 @@ function multishippingMode(it)
 						});
 					},
 					'afterLoad': function(){
```

Comment:
Hi,

With the new version of typewatch, ```this.el``` must be replaced by ```this```.

Best regards